hypervisor/arch/x86/guest/vm.c

```diff
@@ -774,12 +774,22 @@ void prepare_vm(uint16_t vm_id, struct acrn_vm_config *vm_config)
 void launch_vms(uint16_t pcpu_id)
 {
 	uint16_t vm_id;
+	uint64_t pre_vm_cpu_affinity = 0UL;
 	struct acrn_vm_config *vm_config;
 
 	for (vm_id = 0U; vm_id < CONFIG_MAX_VM_NUM; vm_id++) {
 		vm_config = get_vm_config(vm_id);
 		if ((vm_config->load_order == SOS_VM) || (vm_config->load_order == PRE_LAUNCHED_VM)) {
```

Comment:
In this case, will "split this if ( .. == SOS_VM || .. == PRE_LAUNCHED_VM)" to "if (...SOS_VM) .... else {}" simplify the code?

Reply:
sure, will update.